### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created button text classes
- created button background classes
- created button text hover classes
- created button background hover classes
-  created button disabled classes
- created button sizes 
Screen Shot 2022-02-06 at 4 51 05 AM

### CODE REVIEW

File: css/alerts.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  justify-content: space-between;
+}
+
+.alert-btn {
+  position: relative;
+  right: 1rem;
+  background-color: transparent;
+  border: 0;
+  font-size: 1rem;
+}
```
Comment (ashishkatiyar2404) [inline css/alerts.css:41]: naming conventions is really good.

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
UI is amazing. Your code is also good.💯

File: css/alerts.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  height: 3rem;
+  width: 100%;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  height: max-content;
+  display: flex;
+  align-items: center;
+  border-radius: 0.4rem;
+  padding-left: 2rem;
+  gap: 0.8rem;
```
Comment (dhimanaishwarya) [inline css/alerts.css:13]: you can give a shorthand property for padding instead of using padding-top,padding-botton,padding-left

**Comment by rahulyadav139** [comment]
Everything is great to me. UI is eye-catching. Great work.

File: index.html [hunk 3]
```diff
@@ -12,30 +12,153 @@
...
+    <h1 class="text-center pt-10">Disabled Buttons</h1>
+    <div class="pt-10 text-center">
+      <button
+        class="btn btn-square-solid text-white bg-rose-700 btn-bg-hover-rose-600 btn-disabled"
+      >
+        click me
+      </button>
+      <button
+        class="btn text-white btn-bg-hover-teal-500 btn-text-hover-white btn-rounded-solid bg-teal-700 btn-disabled"
+      >
```
Comment (PuneetP16) [inline index.html:86]: very long class list, try to shorten names while keeping readability.

**Review by PuneetP16 — APPROVED** [review]
Good to merge

**Pull request merged**